### DISCUSSION TITLE
Радиусы оружия

### DISCUSSION THREAD

**Discussion by dosinabox** [dosinabox/g2nr_unofficial_update#441] [General]
Всё понятно, кроме длины двуручного Когтя. Нужно ли удлинить его с 120 до 140, чтобы его скриптовый радиус равнялся длине модели? Или это слишком повлияет на баланс?

[Табличка с радиусом](https://docs.google.com/spreadsheets/d/1svDbXAiYe7PLCQ7G0Arjh_njRiDabsRC-9GuLt7APsQ/edit#gid=792497162)

**Comment by Kor-Angar** [comment]
двуручный коготь очень слабый по урону. На мой взгляд, если его сделать, длиннее это хоть как то компенсирует его низкий урон. 

у одноручного Когтя длина же 100 ?

скопировал табличку в гугл таблицу. ссылку поставил в первый пост

**Comment by dosinabox** [comment]
> у одноручного Когтя длина же 100 ?

100. Длина модели - 102,278.

**Comment by Kor-Angar** [comment]
у Орков два меча? :-0


ITMW_2H_ORCSWORD_01 | Краш Варрок | 98 | 100
ITMW_2H_ORCSWORD_02 | Баш Брокдар | 139,953 | 140

**Comment by Kor-Angar** [comment]
может Мачете сделать одинаковой длины?
ITMW_ADDON_HACKER_2H_02 | Старое гигантское мачете | 111,476 | 95 | > | 110
ITMW_ADDON_HACKER_2H_01 | Гигантское мачете | 111,476 | 105 | > | 115

**Comment by Kor-Angar** [comment]
ITMW_TRAINSWORD | Тренировочный меч

откуда он взялся? был в скриптах? у него же ущерба нет? почему с ним тренируются избитый НПС, ведь у него есть ржавый меч / топор?

**Comment by dosinabox** [comment]
> может Мачете сделать одинаковой длины?

Старое хуже попадает, всё правильно)

> откуда он взялся? был в скриптах? у него же ущерба нет? почему с ним тренируются избитый НПС, ведь у него есть ржавый меч / топор?

Добавлен в обновлении для исправления бесконечной генерации ржавых мечей у тренирующихся. Поэтому и цена нулевая.

> у Орков два меча? :-0

Один из них остался из Г1, в Г2 им вооружены люди-ящеры. Всегда хотелось сделать им новое оружие с новой моделью, а это вернуть некоторым оркам.

**Comment by Kor-Angar** [comment]
> Старое хуже попадает, всё правильно)

это тогда штраф на навык надо делать;-) а так два мачете, одно старое(меньше урон), а другое нет(больше урон). одинаковый длины. так радиус почему разный то? ;-)

**Comment by Kor-Angar** [comment]
> Добавлен в обновлении для исправления бесконечной генерации ржавых мечей у тренирующихся. Поэтому и цена нулевая.

так и пусть генерица бесконечно. разве это проблема? :-) или типа кто то их будет продавать что ли? :-) и делать на этом деле деньги? :-)

**Comment by dosinabox** [comment]
> > Старое хуже попадает, всё правильно)
> 
> это тогда штраф на навык надо делать;-) а так два мачете, одно старое(меньше урон), а другое нет(больше урон). одинаковый длины. так радиус почему разный то? ;-)

Скриптовые радиусы и длина моделей теперь будет равна у:
- мачете и старое мачете
- гигантское мачете и старое гигантское мачете

**Comment by dosinabox** [comment]
Скриптовые радиусы и длина моделей теперь равна у всего оружия (с погрешностью +-5 единиц), за исключением:
- дубинка ветра короче на 20 (она просто дурацкая и вообще со штрафом)
- тяжелый сук длиннее на 18 (скорее всего, сделан так разработчиками намеренно)
- оружие орков длиннее на 30-35 (изменения слишком повлияют на баланс)
- священный молот длиннее на 43 (он же магический! и нужен для ближнего боя с големом)

**Comment by Kor-Angar** [comment]
надо будет актуальные данные по радиусам  добавить в вики
